cypress/integration/plugins/security/permissions_spec.js

```diff
@@ -73,13 +73,7 @@ if (Cypress.env('SECURITY_ENABLED')) {
     });
 
     it('should create new action group successfully by selecting `Create from blank`', () => {
-      cy.mockPermissionsAction(
```

Comment:
we should not be loading the action group before creating it. The action-group `test-creation` should not be present on initial page load.